test/powershell/Modules/Microsoft.PowerShell.Utility/RunspaceCmdlets.Tests.ps1

```diff
@@ -0,0 +1,39 @@
+Describe "Get-Runspace cmdlet tests" -Tag "CI" {
+    BeforeAll {
+        # Clear out all unexpected runspaces
+        # id 1 is the one we started with
+        Get-Runspace |?{$_.id -ne 1} | %{$_.dispose()}
+        $CurrentRunspace = $ExecutionContext.Host.Runspace
```

Comment:
tests should clean up after themselves and not pollute, and this code forces that clean up in a rather aggressive manner, however, it may be that it is too aggressive.

---
In reply to: [78446552](https://github.com/PowerShell/PowerShell/pull/2239#discussion_r78446552) [](ancestors = 78446552)